utils/slack.js

```diff
@@ -0,0 +1,57 @@
+import fetch from 'isomorphic-unfetch'
+
+/**
+ * Returns an absolute URL in the case of `getInitialProps`
+ */
+function getOrigin(req) {
+  if (typeof window === 'undefined') {
+    const { host } = req.headers
```

Comment:
This is only safe when deployed to Now, on other hosting providers `host` can be overwritten by  the incoming request.